content/millennium/r4/documents/binary.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Binary | R4 API
+---
+
+# Binary
+
+* TOC
+{:toc}
+
+## Overview
+
+The Binary resource can contain any clinical content such as text, images, and PDFs.  This resource is currently limited to clinical documents and diagnostic reports.  The diagnostic reports are currently limited to Anatomic Pathology and Radiology.
+
+It is recommended to request all Binary resources only after obtaining a link to the resource via references from DiagnosticReport or DocumentReference. It is not recommended to start a workflow in the Binary resource.
+
+The consumer must populate the Accept header with either application/fhir+json or the format returned in the attachment.contentType of the referring resource.  If the Accept header is application/fhir+json, a FHIR Binary resource is returned with the raw data populated in the data. Otherwise, the raw data will be returned (not contained within a FHIR resource). For more information, see [the Binary Documentation](http://hl7.org/fhir/r4/binary.html#rest).
+
+NOTE: To retrieve documents with application/pdf contentType, the client must have installed Cerner’s Clinical Reporting (XR) solution. For additional information about Cerner’s Clinical Reporting (XR) solution see [Clinical Reporting (XR) Reference Pages](https://wiki.ucern.com/display/reference/Clinical+Reporting+XR+Reference+Pages).
```

Comment:
"Note: in order to retrieve documents with the `application/pdf` content type, Cerner's Clinical Reporting (XR) solution must be installed for the client site you are loading data from. For additional information about Cerner's Clinical Reporting (XR) solution see Cerner's [Clinical Reporting (XR) Reference Pages](https://wiki.ucern.com/display/reference/Clinical+Reporting+XR+Reference+Pages)."

Reply:
https://github.com/cerner/fhir.cerner.com/commit/70b171a1701f4afd9b85d8b10f6a8a609678eb5f#diff-aaa0de638fb809d6747b985beb976128R18